cpp/ql/src/JPL_C/LOC-3/Rule 17/BasicIntTypes.ql

```diff
@@ -12,7 +12,8 @@
 import cpp
 
 predicate allowedTypedefs(TypedefType t) {
-  t.getName() = ["I64", "U64", "I32", "U32", "I16", "U16", "I8", "U8", "F64", "F32"]
+  t.getName() = ["I64", "U64", "I32", "U32", "I16", "U16", "I8", "U8", "F64", "F32",
+                 "int64_t", "uint64_t", "int32_t", "uint32_t", "int16_t", "uint16_t", "int8_t", "uint8_t"]
```

Comment:
I think the formatting here breaks the CI job that check the formatting. Easiest way to fix this is to ask VSCode to format the file.